### PR TITLE
qa,doc: drop support of ubuntu trusty

### PR DESCRIPTION
**Pull request by tchaikov**

### CODE REVIEW

File: qa/suites/fs/multiclient/tasks/fsx-mpi.yaml.disabled [hunk 1]
```diff
@@ -1,7 +1,4 @@
 # make sure we get the same MPI version on all hosts
-os_type: ubuntu
-os_version: "14.04"
```
Comment (liewegas) [inline qa/suites/fs/multiclient/tasks/fsx-mpi.yaml.disabled]: should probably just switch this to 16.04.. we still need uniformity?

Comment (tchaikov) [reply qa/suites/fs/multiclient/tasks/fsx-mpi.yaml.disabled]: @liewegas i added a link to `../../../../distros/supported/ubuntu_latest.yaml` in this very commit. so i am switching this to 16.04.

Comment (liewegas) [reply qa/suites/fs/multiclient/tasks/fsx-mpi.yaml.disabled]: i see, :+1:

**Comment by tchaikov** [comment]
- fs suite: http://pulpito.ceph.com/kchai-2017-12-05_05:26:56-fs-wip-kefu-testing-2017-12-05-1155-distro-basic-mira/
- hadoop suite: http://pulpito.ceph.com/kchai-2017-12-05_05:28:21-hadoop-wip-kefu-testing-2017-12-05-1155-distro-basic-mira/ . proposed fix posted at https://github.com/ceph/teuthology/pull/1133

**Pull request merged**